homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -1,16 +1,23 @@
 """Config flow for RFXCOM RFXtrx integration."""
 import logging
+import os
 
+import RFXtrx as rfxtrxmod
+import serial
+import serial.tools.list_ports
```

Comment:
Doesn't this add a new dependency for manifest?

Reply:
I think you are right, I copied this code from `zha` integration which also defines `pyserial` in the manifest.

Reply:
We need to add it if the library doesn't already depend on pyserial.

Reply:
In that case we can skip it, pyRFXtrx depends on pyserial.